src/client/swagger/unity.yml

```diff
@@ -1277,10 +1254,35 @@ components:
         date:
           type: string
           description: date org was created
-        # TODO(ariel): this seems cyclical since accounts and orgs are subsets of one another
-        account:
-          $ref: '#/components/schemas/Account'
-      required: [id, idpeID, name, account, region, provider, date]
+        accountType:
+          type: string
+          description: type of the account
+          enum:
+            - free
+            - cancelled
+            - pay_as_you_go
+        accountBalance:
+          type: number
+          description: balance of the account
+        accountEmail:
+          type: string
+          description: billing contact email
+        accountId:
+          type: string
+          description: id of the account associated with the org
+      required:
+        [
+          id,
+          quartzId,
+          name,
+          accountType,
+          accountBalance,
+          accountEmail,
+          accountId,
```

Comment:
Since most of the `account` data is only used by the operator, i wasn't sure if we wanted to make it required or not